tornado_mysql/converters.py

```diff
@@ -123,7 +123,7 @@ def convert_datetime(obj):
         usecs = '0'
         if '.' in hms:
             hms, usecs = hms.split('.')
-        usecs = float('0.' + usecs) * 1e6
+        usecs = round(float('0.' + usecs) * 1e6)
```

Comment:
You may be right. I haven't checked either! 

I just wanted to leave the smallest footprint that passes for all million possible microsecond test cases.